tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create.go

```diff
@@ -0,0 +1,118 @@
+	"github.com/stretchr/testify/suite"
+)
+
+type CRTBGenTestSuite struct {
+	suite.Suite
+	client               *rancher.Client
+	crtbName             string
+	roleTemplateName     string
+	userPrincipalName    string
+	session              *session.Session
+	cluster              *management.Cluster
+	clusterName          string
+	clusterID            string
+	clusterV1            *v1.Cluster
+	newUser              *management.User
+	crtbJsonTemplateData []byte
+}
+
+const (
+	crtbApiEndPoint                 = "management.cattle.io.clusterroletemplatebinding"
+	localCluster                    = "local"
+	defaultRoleTemplateName         = "cluster-member"
+	clusterOwner                    = "cluster-owner"
+	clusterMember                   = "cluster-member"
+	crtbManage                      = "clusterroletemplatebindings-manage"
+	metadata                        = "metadata"
+	userName                        = "userName"
+	roleTemplateName                = "roleTemplateName"
+	namespace                       = "namespace"
+	name                            = "name"
+	clusterName                     = "clusterName"
+	userPrincipalName               = "userPrincipalName"
+	labels                          = "labels"
+	crtbNameWithTenChar             = 10
+	ID                              = "id"
+	crtbConflictError               = "409 Conflict"
+	crtbAlreadyExisted              = "AlreadyExists"
+	badRequest                      = "400 Bad Request"
+	notFound                        = "404 Not Found"
+	cluserNameAndNameSpaceSameValue = "Forbidden: clusterName and namespace must be the same value"
+)
+
+func (crtb *CRTBGenTestSuite) TearDownSuite() {
+	crtb.session.Cleanup()
+}
+
+func (crtb *CRTBGenTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	crtb.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(crtb.T(), err)
+
+	crtb.client = client
+	crtb.clusterName = client.RancherConfig.ClusterName
+	require.NotEmptyf(crtb.T(), crtb.clusterName, "Cluster name to install should be set")
+
+	crtb.clusterID, err = clusters.GetClusterIDByName(crtb.client, crtb.clusterName)
+	require.NoError(crtb.T(), err, "Error getting cluster ID")
+
+	crtb.crtbJsonTemplateData = createJsonTemplateForClusterRoleTemplateBinding().crtbJsonTemplateData
+	require.NoError(crtb.T(), err)
+
+	crtb.newUser, err = users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(crtb.T(), err)
+}
+
+func createJsonTemplateForClusterRoleTemplateBinding() CRTBGenTestSuite {
+	crtbTemplate := map[string]interface{}{
+		"type":       "clusterRoleTemplateBinding",
+		"apiVersion": "management.cattle.io/v3",
+		"kind":       "ClusterRoleTemplateBinding",
+		metadata: map[string]interface{}{
+			namespace: "",
+			name:      "",
+			labels: map[string]interface{}{
+				"hello": "World",
+			},
+		},
+		clusterName:       "",
+		roleTemplateName:  "",
+		userName:          "",
+		userPrincipalName: "",
+	}
```

Comment:
Not a requested change per say, just a question. 

Should we add something similar to a CRTB struct in extensions for more reuse? 

Maybe here: https://github.com/rancher/rancher/blob/release/v2.8/tests/framework/extensions/clusterrolebindings/clusterrolebindings.go

Something like what was added for config maps: https://github.com/rancher/rancher/blob/release/v2.8/tests/framework/extensions/configmaps/configmaps.go

Unless that route was tried and there was some issues or if it just doesn't make sense?

Otherwise, once this question is answered everything looks good.